src/main/kotlin/com/example/mykku/image/service/ImageUploadService.kt

```
@@ -0,0 +1,106 @@
+    }
+    
+    private fun extractImageDimensions(imageBytes: ByteArray): Pair<Int, Int> {
+        return try {
+            val bufferedImage: BufferedImage = ImageIO.read(ByteArrayInputStream(imageBytes))
+                ?: throw MykkuException(ErrorCode.IMAGE_UNREADABLE)
+            
+            Pair(bufferedImage.width, bufferedImage.height)
+        } catch (e: MykkuException) {
+            // 도메인 예외는 그대로 전달
+            throw e
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)